### PR TITLE
[Rules] Maintenance updates

### PR DESCRIPTION
**Pull request by pedrosousa**
* Updates curl examples.
* Moves repeated content to partial.
* Reviews capitalization.

### CODE REVIEW

File: content/rules/origin-rules/_index.md [hunk 1]
```diff
@@ -30,6 +30,6 @@ The Origin Rule expression will determine when these overrides will be applied.
...
+If you override the hostname with an origin rule (via `Host` header override or DNS record override) and add a header override to your load balancer configuration, the origin rule will take precedence over the load balancer configuration.
 
-Like [Page Rules](/support/page-rules/understanding-and-configuring-cloudflare-page-rules-page-rules-tutorial/), an Origin Rule performing a `Host` header override will update the SNI value of the original request to the same value of the `Host` header. To set an SNI value different from the `Host` header override, add an SNI override in the same Origin Rule or create a separate Origin Rule for this purpose.
+Like [Page Rules](/support/page-rules/understanding-and-configuring-cloudflare-page-rules-page-rules-tutorial/), an origin rule performing a `Host` header override will update the SNI value of the original request to the same value of the `Host` header. To set an SNI value different from the `Host` header override, add an SNI override in the same origin rule or create a separate origin rule for this purpose.
```
Comment (RebeccaTamachiro) [inline content/rules/origin-rules/_index.md:35]:
```suggestion
Like [page rules](/support/page-rules/understanding-and-configuring-cloudflare-page-rules-page-rules-tutorial/), an origin rule performing a `Host` header override will update the SNI value of the original request to the same value of the `Host` header. To set an SNI value different from the `Host` header override, add an SNI override in the same origin rule or create a separate origin rule for this purpose.
```

File: content/rules/origin-rules/faq.md [hunk 2]
```diff
@@ -10,26 +10,26 @@ meta:
 
 Below you will find answers to the most commonly asked questions regarding Origin Rules.
 
-## What happens if I use both an Origin Rule and a Page Rule to perform a Host header/DNS record override?
+## What happens if I use both an origin rule and a Page Rule to perform a Host header/DNS record override?
```
Comment (RebeccaTamachiro) [inline content/rules/origin-rules/faq.md]:
```suggestion
## What happens if I use both an origin rule and a page rule to perform a Host header/DNS record override?
```

File: content/rules/origin-rules/faq.md [hunk 3]
```diff
@@ -10,26 +10,26 @@ meta:
...
 
-In this situation the Origin Rule parameters will override the [Page Rule](/support/page-rules/understanding-and-configuring-cloudflare-page-rules-page-rules-tutorial/) parameters. Consider the following example scenarios:
+In this situation the origin rule parameters will override the [Page Rule](/support/page-rules/understanding-and-configuring-cloudflare-page-rules-page-rules-tutorial/) parameters. Consider the following example scenarios:
```
Comment (RebeccaTamachiro) [inline content/rules/origin-rules/faq.md]:
```suggestion
In this situation the origin rule parameters will override the [page rule](/support/page-rules/understanding-and-configuring-cloudflare-page-rules-page-rules-tutorial/) parameters. Consider the following example scenarios:
```

File: content/rules/origin-rules/faq.md [hunk 4]
```diff
@@ -10,26 +10,26 @@ meta:
...
 
-* A Page Rule defines a Host header override, but not a resolve override (or DNS record override). An Origin Rule defines a DNS record override, but not a Host header override. The resulting request will have the `Host` header defined by the Page Rule and the origin hostname defined by the Origin Rule.
-* A Page Rule defines a Host header override, and an Origin Rule also defines a Host header override. The resulting request will have the `Host` header defined by the Origin Rule.
+* A Page Rule defines a Host header override, but not a resolve override (or DNS record override). An origin rule defines a DNS record override, but not a Host header override. The resulting request will have the `Host` header defined by the Page Rule and the origin hostname defined by the origin rule.
+* A Page Rule defines a Host header override, and an origin rule also defines a Host header override. The resulting request will have the `Host` header defined by the origin rule.
```
Comment (RebeccaTamachiro) [inline content/rules/origin-rules/faq.md]:
```suggestion
* A page rule defines a Host header override, but not a resolve override (or DNS record override). An origin rule defines a DNS record override, but not a Host header override. The resulting request will have the `Host` header defined by the page rule and the origin hostname defined by the origin rule.
* A page rule defines a Host header override, and an origin rule also defines a Host header override. The resulting request will have the `Host` header defined by the origin rule.
```

File: content/rules/custom-error-responses/create-api.md [hunk 5]
```diff
@@ -3,48 +3,44 @@ title: Create via API
...
-  title: Create custom error responses via API
-layout: list
+  title: Create a custom error response via API
 ---
 
-# Create via API
+# Create a custom error response via API
 
-Use the [Rulesets API](/ruleset-engine/rulesets-api/) to create custom error responses via API. Define the response configuration in the `action_parameters` field.
+You configure custom error responses as [rules](/ruleset-engine/about/rules/) belonging to the `http_custom_errors` phase. Use the [Rulesets API](/ruleset-engine/rulesets-api/) to create custom error responses via API.
```
Comment (RebeccaTamachiro) [inline content/rules/custom-error-responses/create-api.md]:
```suggestion
Configure custom error responses as [rules](/ruleset-engine/about/rules/) belonging to the `http_custom_errors` phase. Use the [Rulesets API](/ruleset-engine/rulesets-api/) to create custom error responses via API.
```

File: content/rules/origin-rules/create-api.md [hunk 6]
```diff
@@ -193,7 +182,7 @@ header: Response
           }
         },
         "expression": "starts_with(http.request.uri.path, \"/hr-app/\")",
-        "description": "Origin Rule for the company's HR application",
+        "description": "Origin rule for the company's HR application",
```
Comment (RebeccaTamachiro) [inline content/rules/origin-rules/create-api.md]:
```suggestion
        "description": "Origin rule for the company HR application",
```

Comment (RebeccaTamachiro) [reply content/rules/origin-rules/create-api.md]: Seems like after changing from `json` to `bash`, having `'s` within the example caused formatting to be inconsistent. Font color after `'` turns white.

**Review by RebeccaTamachiro — APPROVED** [review]

File: content/rules/origin-rules/_index.md (hunk 1 above)
Comment (pedrosousa) [reply content/rules/origin-rules/_index.md:35]: Here Page Rules is the feature name. I'll address the Page Rules review in a separate PR, and check whether it makes sense to keep it title cased when referring to the feature (I believe it does).

File: content/rules/origin-rules/create-api.md (hunk 6 above)
Comment (pedrosousa) [reply content/rules/origin-rules/create-api.md]: I believe that here it's fine (we're in the response, which still uses `json`). However, the request was in fact broken due to the `'`. To simplify the required changes, I'll also update the request according to your suggestion.

File: content/rules/origin-rules/create-api.md [hunk 7]
```diff
@@ -141,21 +130,21 @@ $ curl -X PUT \
...
+curl --request PUT \
+https://api.cloudflare.com/client/v4/zones/{zone_id}/rulesets/{ruleset_id} \
+--header "Authorization: Bearer <API_TOKEN>" \
+--header "Content-Type: application/json" \
+--data '{
   "rules": [
     {
       "expression": "starts_with(http.request.uri.path, \"/hr-app/\")",
-      "description": "Origin Rule for the company's HR application",
+      "description": "Origin rule for the company's HR application",
```
Comment (pedrosousa) [inline content/rules/origin-rules/create-api.md]:
```suggestion
      "description": "Origin rule for the company HR application",
```

**Pull request merged**